backend/Wheelingful.BLL/Services/Chapters/ChapterAuthorService.cs

```diff
@@ -6,7 +6,9 @@
 
 namespace Wheelingful.BLL.Services.Chapters;
 
-public class ChapterAuthorService(WheelingfulDbContext dbContext) : IChapterAuthorService
+public class ChapterAuthorService(
+    IChapterTextService chapterText, 
```

Comment:
add Service suffix, otherwise it sounds like a string

Reply:
Good point. Will do.